src/Pretzel.Logic/Commands/BaseParameters.cs

```diff
@@ -21,6 +21,8 @@ public sealed class BaseParameters
         [Export("SourcePath")]
         public string Path { get; private set; }
 
+        public string Configuration { get; private set; }
```

Comment:
Since the parameter has nothing to do with configuration but but seems related to the environment with your example, could you rename it `Environment`?
The `Configuration` parameter would be more suited for defining the path of a custom configuration file.